### PR TITLE
created html pages

### PR DESCRIPTION
**Pull request by sansadev**

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  margin-left: 2.5rem;
+ }
+  li {
+  float: left;
+
+ }
+li  a {
+  display: block;
+  text-align: center;
+  padding: 14px 16px;
```
Comment (MadEste) [inline style/style.css]: Not an issue for now, but make sure you start using % or em/rem units moving forward instead of px. px are not a responsive unit.

**Comment by sansadev** [comment]
Hi,

I am completely aware about this. I was frustrated trying to do multiple
ways for cards. Also I am not still so satisfied with header. So I tried
on/off codes on chrome Dev tools. I used px to test and see immediate
change. After that converted all px to rems, but left in the ul. My mistake
Yes... I will be careful next time
Thank you so much for your help.

Best
Swati

File: style/style.css [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+  padding:1.8rem;
+ }
+  li {
+  float: left;
+
+ }
+li  a {
+  display: block;
+  text-align: center;
+  padding: 14px 16px;
```
Comment (MadEste) [inline style/style.css]: Nice work changing the px elsewhere. Make sure to update line 83 as well.

File: style/style.css [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+.container {
+  height:15.625rem;
+  padding: 0.875rem  3rem;
+}
+.container h3 {
+  font-size: 1.5rem;
+  color: #094160;
+  margin:0  1rem 1rem 0;
+}
+.container p {
```
Comment (MadEste) [inline style/style.css:140]: Text is overflowing at narrower resolutions. Try playing around with sizing and overflow for better results.

File: style/style.css [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+}
+
+.card-content {
+  border: 0.125rem solid gray;
+  width: 85%;
+  box-shadow:0 0.5rem 0.625rem 0 #888888;
+  margin: 2rem;
+}
+
+.container {
```
Comment (MadEste) [inline style/style.css]: Padding is a little large at different resolutions, Try experimenting with smaller values.

**Comment by sansadev** [comment]
Hi,

trying to keep three elements on a row as per design. but i m missing
something. van u help please?
here is the screen shot.

**Comment by sansadev** [comment]
i might be able to do it if i can use div element inside form. is that ok
to do?

File: js/demo.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+   let result = x *  x ;
+
+   alert(result);
+ }
+
+
+function multiplier() {
+  let input1 = prompt("Please enter a number");
+  let x = Number(input1);
+  let input2 = prompt("Please enter another number");
```
Comment (MadEste) [inline js/demo.js:18]: Nice work! You can also chain these functions together:
`let x = Number(prompt("Please enter a number"));`

File: js/index.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+
+let totalPrice = 0;
+function dogInfo(name, breed, price) {
+  let message = name + ' ,' +  breed + ', '+  price;
+  alert(message);
+}
+
+function addToCart(price) {
+  totalPrice += price;
+alert(`Total Amount is:  ${totalPrice}` );
```
Comment (MadEste) [inline js/index.js]: Nice work!

File: index.html [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+      <li><a href="dogs.html">Dogs</a></li>
+      <li><a href="contact.html">Contact</a></li>
+    </ul>
+  </nav>
+  <div class="copyright-footer">
+     © 2018 Adoptadog, LLC | Seattle, WA
+  </div>
+
+  </footer>
+<script src="js/index.js"></script>
```
Comment (MadEste) [inline index.html]: 👍

File: style/style.css [hunk 8]
```diff
@@ -1,24 +1,54 @@
...
+  color:#81b741;
+  font-size:1.875rem;
+  position:absolute;
+  margin-left:6rem;
+  word-spacing:0.5rem;
+
+}
+.main-header {
+  width:100%;
+  height:500px;
```
Comment (MadEste) [inline style/style.css]: This is poor responsive design, you should not be using px to size elements.

File: style/style.css [hunk 9]
```diff
@@ -1,24 +1,54 @@
...
+.headerImg {
+  margin-left: 4rem;
+  width:40%;
+  margin-top: 8rem;
+
+}
+#header-seattle-agency {
+  color:#81b741;
+  font-size:1.875rem;
+  position:absolute;
```
Comment (MadEste) [inline style/style.css]: I'm not sure you need position: absolute here. and often its bad practice.

**Comment by sansadev** [comment]
Hi,
yes I will take care of this.

I am almost done with lesson-8. it was pretty interesting. But I dont know
where to create index.html n main.js in order to make dynamic blog-posts.
should i create a simple html and simple js file inside adopt a dog
folder(existing)? Or i need to mke a different folder and a new repo?
I hv done this assignment in a separate folder on my laptop
Best
Swati

File: js/main.js [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+const p2 = document.createElement('p');
+p2.textContent = blogContainer.p2;
+parablog.appendChild(p2);
+
+divblog.appendChild(parablog);
+div.appendChild(divblog);
+}
+
+/
```
Comment (MadEste) [inline js/main.js]: stray '/' causing error

File: js/main.js [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+const div = document.getElementsByClassName('flex-container')[0];
+// const h2 = document.createElement('h2');
+// h2.textContent = "Travelling with your dog";
+// div.appendChild(h2);
+// const img = document.createElement('img');
+// img.setAttribute('src','file:///C:/Users/swati/OneDrive/Desktop/html200/adopt-a-dog/images/blog-1.jpg');
+// div.appendChild(img);
+for (let i = 0; i < blogContainers.length; i++) {
+const blogContainer = blogContainers[i];
```
Comment (MadEste) [inline js/main.js:33]: Would recommend you tab inside of {} to make code easier to read.

File: js/main.js [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+const divblog = document.createElement('div');
+divblog.setAttribute('class', 'blog-container');
+const img = document.createElement('img');
+img.setAttribute('src', 'file:///C:/Users/swati/OneDrive/Desktop/html200/adopt-a-dog/images/blog-1.jpg');
```
Comment (MadEste) [inline js/main.js]: Don't use a hardcoded path, this may work locally but won't online.

File: contact.html [hunk 13]
```diff
@@ -0,0 +1,153 @@
...
+      <p>Cost to Adopt: $123.45</p>
+    </div>
+
+  </div>
+  <div class="total-adoption">
+    <h2>Total Adoption Cost</h2>
+    <p>$1234.56</p>
+  </div>
+  <div class="checkout-button">
+    <button type="submit" name="button" id="btn" onclick="getInfo()">submit</button>
```
Comment (MadEste) [inline contact.html]: changing type="submit", to type="button" should keep the button from submitting the form and refreshing the page.

File: js/form.js [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+  form.name = document.getElementById('name').value;
+  form.email = document.getElementById('email').value;
+  form.street = document.getElementById('street').value;
+  form.city = document.getElementById('city').value;
+  form.state = document.getElementById('state-select').value;
+  form.zip = document.getElementById('zip').value;
+  form.pickup = document.getElementById('pickup').value;
+
+  const radio = document.getElementsByName('firsttimeadopter');
+  for(let i=0; i<radio.length; i++ ) {
```
Comment (MadEste) [inline js/form.js:39]: Nice way of making checkbox legible.

**Pull request closed without merging**